setup.py

```diff
@@ -52,26 +54,48 @@
 )
 
 
-py_version = sys.version[:3]
+py_version = sys.version_info
+is_pypy = "PyPy" in sys.version
 
 SCRIPT1 = 'nose2'
-SCRIPT2 = 'nose2-%s' % (py_version,)
+SCRIPT2 = 'nose2-%s.%s' % (py_version.major, py_version.minor)
+
 
+def _get_specific_requirements():
```

Comment:
This function should probably take version as input and return a list of requirements instead using and setting variables outside of its scope.

Also would be nice to have a docstring, even a oneliner.